source/ui.py

```diff
@@ -9,9 +9,54 @@
 See L{gui} for the graphical user interface.
 """
 
+import os
+import sys
+from ctypes import windll, byref, POINTER, addressof
+from comtypes import IUnknown
+from comtypes import automation 
+from logHandler import log
+import gui
 import speech
 import braille
 
+# From urlmon.h
+URL_MK_UNIFORM = 1
+
+# Dialog box properties
+DIALOG_OPTIONS = "dialogWidth:350px;dialogHeight:140px;resizable:yes;center:yes;help:no"
+
+#dwDialogFlags for ShowHTMLDialogEx from mshtmhst.h
+HTMLDLG_NOUI = 0x0010 
+HTMLDLG_MODAL = 0x0020 
+HTMLDLG_MODELESS = 0x0040 
+HTMLDLG_PRINT_TEMPLATE = 0x0080 
+HTMLDLG_VERIFY = 0x0100 
+
+def browseableMessage(message,title=None , isHtml=False):
+	"""Present a message to the user.
+	The message will be presented in an HTML document so that the user can read it with browse mode.
+	@param message: The message in either html or text.
+	@type message: str
+	@param title: The title for the message.
+	@type title: str
+	@param isHtml: Whether the message is html
+	@type isHtml: boolean
+	"""
+	htmlFileName  = os.path.realpath( 'message.html' )
+	if not os.path.isfile(htmlFileName ): 
+		raise LookupError(htmlFileName )
+	moniker = POINTER(IUnknown)()
+	windll.urlmon.CreateURLMonikerEx(0, unicode( htmlFileName ) , byref(moniker), URL_MK_UNIFORM)
+	if not title:
+		# Translators: The default title for the Browseable Message if title arguement is none for browseableMessage.

```

Comment:
nit: For future reference, a translators comment shouldn't reference stuff specific to the code; i.e. "if title arguement is none for browseableMessage". Many translators aren't familiar with the code at all. Since this is so minor, I'll just apply this myself.